Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -123,7 +123,18 @@ export function runQTest(args: QTestArguments): Result {
         const qTestContextInfoFile = Environment.getFileValue("[Sdk.BuildXL]qtestContextInfo");
         qTestContextInfoPath = qTestContextInfoFile.path;
     }
+     
+    let changeAffectedInputListWrittenFile = undefined;

```

Comment:
Could you rename this to ```fileWithImpactedTargets```?

Reply:
I will change the file name, but leave the argument's name as it is, since it is a contract with the engine. I can't change the argument's name here.